### DISCUSSION TITLE
Welcome to nvimdots Discussions!

### DISCUSSION THREAD

**Discussion by ayamir** [ayamir/nvimdots#271] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BitterDarker** [comment]
需要帮助：
We've released version v3.0.0!
Visit https://github.com/ayamir/nvimdots/releases to see the release notes.
If you have icons that can't be rendered correctly (e.g., 𑨩  and � ) or icons with incorrect size, be sure to read this!

To silence this message, remove it from `init.lua` at the config's root directory.

To check the glyphs size, make sure the following icons are very close to the crosses but there is no overlap:
XXXXXXXXX

**Comment by BitterDarker** [comment]


**Reply by ayamir** [reply]
`LspInfo`

**Reply by BitterDarker** [reply]
大佬 有手顺么 百度没找到操作方法

**Reply by ayamir** [reply]
`<leader>li`之后会出现`LspInfo`的悬浮窗口。
这个现象大概率是`copilot`出了问题，按照wiki里面写的，配置好nvm和nodejs版本再试试。
或者直接注释掉`copilot`相关的配置：
https://github.com/ayamir/nvimdots/blob/main/lua/modules/plugins/completion.lua#L57-L68

**Reply by BitterDarker** [reply]
null-ls: require("null-ls.health").check()

- ERROR black: the command "black" is not executable.
- ERROR clang_format: the command "clang-format" is not executable.
- OK prettier: the command "prettier" is executable.
- OK stylua: the command "stylua" is executable.

大佬 运行checkhealth 有这个地方报错 
还有 gd gD 等操作  method textDocument/definition is not supported by any of the servers registered for the current buffer 会出现这种提示

**Reply by ayamir** [reply]
checkhealth中的ERROR是因为网络原因所以这些依赖没有被下载
lsp相关的操作失效也是因为没有启动对应的lsp server，我猜也是因为网络原因没下载成功。

**Comment by BitterDarker** [comment]
大佬 关于复制粘贴不用鼠标 除了每次输入 set mouse=  有没有配置文件永久关闭的方式

**Reply by wxzvim** [reply]
我也想知道，，有没有永久关闭的方式？

**Reply by GaugeAndGravity** [reply]
wiki是一点不看啊
